app/templates/events/createEvent.html

```diff
@@ -237,9 +237,13 @@ <h1 id="pageTitle">{{page_title}}</h1>
           <label class="form-label fw-bold">Invite Cohort(s)</label>
           {% for year,students in bonnerCohorts.items()|reverse %}
           <div class="form-check">
-            <input class="form-check-input" type="checkbox" value="{{year}}" name="cohorts[]" id="cohort-{{year}}">
-            <label class="form-check-label text-truncate w-100" for="cohort-{{year}}">{{year}}-{{year+1}}
-              <span class="text-muted">({{students|map(attribute='fullName')|join(", ")}})</span>
+            <input class="form-check-input" type="checkbox" value="{{year}}" name="cohorts[]" id="cohort-{{year}}"
+            {{ 'checked' if invited_years and year|string in invited_years }}>
```

Comment:
If this is checked we should probably disable it so we aren't allowing people to uncheck it since we don't have the functionality to do anything then.